api/graphql/schema.graphql

```diff
@@ -822,6 +822,15 @@ type WorkExperience implements Experience {
   classification: Classification @with(relation: "classification")
   department: Department @with(relation: "department")
   workStreams: [WorkStream!] @with(relation: "workStreams")
+  supervisoryPosition: Boolean
+  supervisedEmployees: Boolean
+  supervisedEmployeesNumber: String
+  budgetManagement: Boolean
+  annualBudgetAllocation: String
+  seniorManagementStatus: Boolean
+  cSuiteRoleTitle: LocalizedCSuiteRoleTitle
```

Comment:
Why are you renaming this attribute?  It is called `cSuiteRoleTitle` in the model, right?

Reply:
🤷 Followed the same pattern of similar fields in the type, such as:

https://github.com/GCTC-NTGC/gc-digital-talent/blob/a053948f3ae6b782185073e7a9aea672cbf861d0/api/graphql/schema.graphql#L820-L821

Reply:
Weird, I'm not sure how it's working but it does seem to work.  :shrug: